src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,12 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String letterNumber = text.replaceAll("\\W", "").toLowerCase();
+        boolean flagCheck = true;
+        for (int i = 0; i < letterNumber.length() / 2; i++) {
```

Comment:
we can use method reverse() to simplify this code

Reply:
done